WcaOnRails/lib/results_validators/persons_validator.rb

```diff
@@ -111,6 +114,25 @@ def validate(competition_ids: [], model: Result, results: nil)
           if without_wca_id.select { |p2| p2.name == p.name }.length > 1 && !duplicate_newcomer_names.include?(p.name)
             duplicate_newcomer_names << p.name
           end
+          # Look for obvious person name issues
+          splt_name = p.name.split
```

Comment:
Typo: `splt` --> `split` (throughout the entire set of checks, because the variable needs to be renamed everywhere)

Reply:
Fixed!